src/commonMain/kotlin/com/airthings/lib/logging/facility/FileLoggerFacility.kt

```diff
@@ -132,7 +132,7 @@ class FileLoggerFacility(
 
     init {
         coroutineScope.launch {
-            if (!io.isDirectory(baseFolder) && !io.mkdirs(baseFolder)) {
+            if (!io.mkdirs(baseFolder)) {
```

Comment:
I got a bit confused here because `File.mkdirs` returns false when the directory exists, `io.mkdirs` return true for the same. I know it's not the same API, but I had to go to the documentation to get some clarity. So yeah, just putting it here.

Reply:
Yeah, indeed. It's a good excuse to add a comment about it! I'll add the comment before merging.

Reply:
✅